docs/guides/es2k/installing-acc-sdk.md

```diff
@@ -0,0 +1,98 @@
+# Installing the ACC SDK
+
+## Overview
+
+The AARCH64 Software Development Kit (SDK) allows you to use an X86 host
```

Comment:
x86 host?

Reply:
Taking a leaf from your book (looking it up in Wikipedia), I see that they're using **x86** as their canonical form. That, plus the convention of using lower-case **x** to mean "this part's variable", makes for a strong argument. **x86** it is.